legal-api/src/legal_api/services/business_details_version.py

```diff
@@ -812,6 +812,9 @@ def resolution_json(resolution_revision) -> dict:
     @staticmethod
     def business_revision_json(business_revision, business_json):
         """Return the business revision as a json object."""
+        if not business_revision:
+            return business_json
```

Comment:
shouldn't need this.  Should never be null. 

I think it has to do with this [function](https://github.com/bcgov/lear/blob/1b40e6d07ba8b1d090b4ffd21ca3aaef102f1533/legal-api/src/legal_api/services/business_details_version.py#L222-L238) being incorrect.